cumulus/variables.tf

```diff
@@ -152,6 +152,16 @@ variable "urs_client_password" {
 
 # Optional
 
+variable "urs_tea_client_id" {
+  type = string
+  default = null
+}
+
+variable "urs_tea_client_password" {
+  type = string
+  default = null
+}
+
```

Comment:
Can we move these down to where the other TEA variables are?

Reply:
Maybe add a description saying that they will default to the non-tea version of the variables?